GX-TimeFrequency/src/minimax_tau.F90

```diff
@@ -2921,10 +2925,24 @@ subroutine get_points_weights_tau(grid_size, e_range, ac_we, ierr)
        allocate(aw%aw_erange_matrix(2*grid_size, bup+1))
        call set_aw_array(grid_size, aw)
 
+
        ! Select energy region with binary search
        ien = find_erange(bup, aw%energy_range, e_range)
+
+       ! Scale grids for large sizes when erange falls in the
```

Comment:
Giving some reference to why one is doing this would be helpful. Even if it is empirical, add a comment or reference